### DISCUSSION TITLE
USTC Computer Networking:A Top-Down Approach - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#223] [Announcements]
# USTC Computer Networking:A Top-Down Approach - CS自学指南

CS自学指南

https://csdiy.wiki/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BD%91%E7%BB%9C/topdown_ustc/

**Comment by pencilCool** [comment]
书没啃下来，看视频容易抓到重点，很多例子是举的国内的，比较亲切

**Comment by derecknowayback** [comment]
神中神！一定要先看书预习再看视频，看完视频再看一遍书，质量绝绝子！

**Comment by imabackstabber** [comment]
不是你们在说什么绝绝子哦，搞得我好想看

**Reply by teamwong111** [reply]
逮到！

**Reply by imabackstabber** [reply]
> 逮到！

哦牛批，还有这种人的

**Comment by linknight423** [comment]
非常不错的教程，大四一整年能让我静下心来钻研，看书，非常值得

**Comment by Hoooao** [comment]
没有作业吗

**Reply by derecknowayback** [reply]
看完去做CS144和看看小林coding的图解计算机网络，很赞

**Reply by eightyun** [reply]
https://www.educoder.net/paths/zecl9i6m
 可以做一下这个

**Comment by bartorat** [comment]
神中神，老师很多评论的问题都会回复

**Comment by Cambridgedad** [comment]
有幸上过郑老师的课

**Comment by RetenQ** [comment]
神

**Comment by smalleyes-bilingbiling** [comment]
全英文。。听/看不懂。

**Reply by chenfs12138** [reply]
郑坚老师讲的不是中文吗？你在搞笑呢？

**Reply by px341** [reply]
中科大和马萨诸塞大学这两个实验的评论区似乎是合并在一起的

**Comment by mrxmxm** [comment]
what's this?

**Comment by Kin-Zhang** [comment]
感动啊 老师真的 还在回答评论区的问题哎！！！

**Comment by wang-zhuoran** [comment]
本科时候计算机网络的lab就是借鉴UMass的，体验很不错，毕竟能体会到intuition，但的确编程方面有些缺乏了

**Comment by feng503** [comment]
https://pan.baidu.com/s/1KCEdzrBhivGPsxMKvCGlAg

**Reply by feng503** [reply]
提取码：vbmq

**Reply by PatrickYuzhiji** [reply]
感谢感谢～～

**Reply by wcool1** [reply]
感谢，best wishes to you

**Comment by lemonedtree** [comment]
可是这个视频还没有做完欸，56只有一半，78就没有了，我好难受！

**Comment by DeathGun5201** [comment]
书中第一章提到的配套的Web网站上的Java小程序我没有找到，在哪里找啊

**Reply by DeathGun5201** [reply]
找到了，都在上面汇总的资料里

**Comment by TephrocactusMYC** [comment]
这课我愿意称之为华语世界最强！特别清楚！神中神！

**Comment by upupnoah** [comment]
开始啃！

**Comment by iamazy** [comment]
开始啃

**Comment by new-I** [comment]
已经学完第一章了，预计十月底学完，有人一起讨论吗，拉我进群

**Comment by SniffyPP** [comment]
如果说学校的课程教材是谢希仁的《计算机网络》，可以看这个课程吗？

**Reply by Asxjdb** [reply]
谢希仁的思路是自底向上，推荐湖科大教书匠的：https://www.bilibili.com/video/BV1NT411g7n6。

**Reply by SniffyPP** [reply]
谢谢你~

**Comment by willsherlock** [comment]
如果想要参加网安竞赛，看完这几章课还有什么课程推荐吗？

**Comment by yi427** [comment]
超喜欢的一门课程，老师讲的很用心

**Comment by goahead656** [comment]
请问有人解决字母的问题了吗？有的视频无法显示字幕哎

**Comment by goahead656** [comment]
[B站链接-有英文汉语双字幕-可以安心食用](https://www.bilibili.com/video/BV1mb4y1d7K7/?spm_id_from=333.337.search-card.all.click&vd_source=33e0117567559c38b0dd6967cb88232b)

**Comment by GongNanyue** [comment]
这个也很好
https://www.bilibili.com/video/BV1c4411d7jb/

**Reply by SMHILYZYH** [reply]
这个感觉有点偏向应试

**Comment by goahead656** [comment]
[lab的中英文翻译，包括lab的solutions](https://github.com/moranzcw/Computer-Networking-A-Top-Down-Approach-NOTES?tab=readme-ov-file)

**Comment by 1Sunjiaqi** [comment]
建议先看一章书再看视频，书写的很好很细，课堂跳跃，如果没看过书，很多老师只是提一嘴过去的知识点自己很难注意到和理解

**Comment by pluto0x0** [comment]
课程课件有第八章网络安全的吗？

**Comment by asdfefbhjy** [comment]
Computer-Networking-A-Top-Down-Approach-8th-Edition , 正在啃书, 这书写得太好了, 尤其是第三章的 Section 3.1-3.4,   一点点增加各种机制, 用状态机来描述协议, 简直完美, 原话是 " see how these mechanisms were incrementally added to cover increasingly complex (and realistic) models of the channel connecting the sender and receiver, or to improve the performance of the protocols",  这几小节学下来简直是一种享受 !!! 强烈建议一定要读课本!!!

**Reply by asdfefbhjy** [reply]
建议读英文原版, 读中文版的会增加一层理解难度和记忆成本

**Comment by LiAi16** [comment]
课后习题有答案吗，包括那些作业

**Comment by AcuLY** [comment]
这本书的原版写的真的很易读，对比下来中文版的翻译太灾难了

**Comment by hxxxi-malog** [comment]
这门课确实是中文互联网为数不多的精品免费课

**Comment by 1010fzp0101** [comment]
我前七章浅读了一遍， 第八章关于密码学的并没有怎么读， 然后准备过段时间开始第二遍学习， 有人一起交流的吗？
